tutorials/RandomNumberGeneration/ReferenceImplementation.qs

```diff
@@ -48,7 +44,8 @@ namespace Quantum.Kata.RandomNumberGeneration {
     }
 
     // Exercise 5.
-    operation RandomNumberInRange_Reference (min : Int, max : Int) : Int {
+    operation RandomNumberInRange_Reference (range : (Int,Int)) : Int {
```

Comment:
You don't need to change the signature of the operation; Q# assumes that any set of parameters is a tuple, and converts it to/from a tuple automatically. (That's why the build passes even though the signature in Tasks.qs is still two integers!)

Reply:
Done.